### DISCUSSION TITLE
Does UpdateHub collect any Data? (GDPR / DSGVO)

### DISCUSSION THREAD

**Discussion by schittli** [NexovaDev/UpdateHub#293] [Q&A] [answered]
Hello

**thanks a lot for all doing and sharing the great work with UpdateHub!**

As we have GDPR (in the EU, it is DSGVO), I have a probably small question:
**Does UpdateHub collect any Data and transfer it to the internet?**

Thanks a lot, kind regards,
Thomas

**Comment by onyx-nxt** [comment, marked as answer]
Hi,
We do not collect, transmit, or store any personally identifiable information. More information can be found [here](https://github.com/KK-Designs/UpdateHub/blob/main/Privacy%20Policy.md "Privacy Policy")

**Reply by schittli** [reply]
Hello @KK-Designs 
thank you very much for your answer!
